releases.md

```diff
@@ -0,0 +1,22 @@
+
+# .NET Releases
+
+The .NET team releases new versions of .NET annually, each November. Releases are either Long Term Support (LTS) or Current, our two [release types](release-notes/release-types.md).
+
+The following table lists currently in-support .NET releases, per [Microsoft support policy](microsoft-support.md).
+
+|  Version  |  Release Date | Support Level | Supported Patch Version | End of Support |
+| :-- | :-- | :-- | :-- | :-- |
+| [.NET 5.0](https://dotnet.microsoft.com/download/dotnet-core/5.0) | [November 10, 2020](https://devblogs.microsoft.com/dotnet/announcing-net-5-0/) | Current | [5.0.1](release-notes/5.0/README.md) | February 10, 2022 |
```

Comment:
I'm not crazy about stating exact dates when they are not certain.  How about simply saying the EOS date is `February, 2022` with a possible indicator that this is an estimate?

Reply:
Fixed.